charts/pega/values-large.yaml

```diff
@@ -183,7 +183,7 @@ global:
           # For GKE Pre-shared Certificate, use ingress.gcp.kubernetes.io/pre-shared-cert: <pre-shared-certificate-name>
           # For GKE to use static IP for load balancer, use kubernetes.io/ingress.global-static-ip-name: <global-static-ip-name>
           ssl_annotation:
-          # for enabling TLS in Openshift Route as Reencrypt, add certificate, key and CaCertificate PEM or crt format.
+          # For Openshift, to enable TLS, the termination policy would be reencrypt rather than edge, and we can add our certificate, the corresponding key using the appropriate .pem or .crt format and specify a CA certificate to validate the endpoint certificate, securing the connection from the browser to the router.
```

Comment:
can we refine this??
For Openshift, Pega  deployments enable TLS to secure the connection from the browser to the router by creating the route using reencrypt termination policy. Add your certificate, the corresponding key using the appropriate .pem or .crt format and specify a CA certificate to validate the endpoint certificate.